vignettes/estimate_ascertainment.Rmd

```diff
@@ -0,0 +1,279 @@
+
+We filter for data from the United Kingdom here, while later examples will use more of the data.
+
+```{r}
+# get Covid data provided with the package
+data("covid_data")
+
+# filter for the U.K
+df_covid_uk <- filter(covid_data, country == "United Kingdom")
+
+# view the data format
+tail(df_covid_uk)
+```
+
+We then subset the data to focus on just the first few months of the outbreak in the United Kingdom, and plot the cases and deaths to explore the data.
+
+```{r}
+# the first few months of the pandemic
+df_covid_uk_subset <- filter(df_covid_uk, date <= "2020-05-31")
+```
+
+```{r fig.cap="Incidence of cases over time for the ongoing COVID-19 outbreak in the U.K.", class.source = 'fold-hide'}
+ggplot(df_covid_uk_subset) +
+  geom_step(
+    aes(
+      x = date, y = cases
+    ),
+    colour = "steelblue"
+  ) +
+  scale_x_date(
+    date_labels = "%b-%Y"
+  ) +
+  scale_y_continuous(
+    labels = comma
+  ) +
+  labs(
+    x = "Date", y = "Cases"
+  ) +
+  coord_cartesian(
+    expand = FALSE
+  ) +
+  theme_classic() +
+  theme(legend.position = "top")
+```
+
+```{r fig.cap="Incidence of deaths over time for the ongoing COVID-19 outbreak in the U.K.", class.source = 'fold-hide'}
+ggplot(df_covid_uk_subset) +
+  geom_step(
+    aes(
+      x = date, y = deaths
+    ),
+    colour = "brown"
+  ) +
+  scale_x_date(
+    date_labels = "%b-%Y"
+  ) +
+  scale_y_continuous(
+    labels = comma
+  ) +
+  labs(
+    x = "Date", y = "Deaths"
+  ) +
+  coord_cartesian(
+    expand = FALSE
+  ) +
+  theme_classic() +
+  theme(legend.position = "top")
+```
+
+### Onset-to-death distribution for Covid-19
+
+We obtain the appropriate distribution reported in @linton2020; this is a log-normal distribution with $\mu$ = 2.577 and $\sigma$ = 0.440.
+
+### Estimating the proportion of cases that have been ascertained
+
+We use the `estimate_ascertainment()` function to calculate the time-varying CFR for the COVID-19 epidemic in the U.K.
+
+The function includes a `type` argument, which determines whether `cfr_static()` or `cfr_time_varying()` is used to estimate the delay-adjusted severity of the disease.
+
+The ascertainment ratio is calculated as the disease severity calculated from the data, divided by the 'known' disease severity; this is expected to be known or assumed from our best knowledge of the pathology of the disease.
+
+This known disease severity is passed to the `severity_baseline` argument in `estimate_ascertainment()`, and forms the numerator in the resulting under-ascertainment calculation.
+
+We assume that the 'true' CFR of Covid-19 is 0.014 (i.e. 1.4%) [@verity2020]. Future plans for this package include ability to incorporate uncertainty in CFR estimates when calculating under-ascertainment.
+
+The other arguments are the same as those found in `cfr_time_varying()`. 
+
+```{r }
+df_reporting_static <- estimate_ascertainment(
+  data = df_covid_uk_subset,
+  delay_density = function(x) dlnorm(x, meanlog = 2.577, sdlog = 0.440),
```

Comment:
should be discretised?

Reply:
Will leave for now per main thread discussion